jena-arq/pom.xml

```diff
@@ -216,6 +216,7 @@
             </group>
           </groups>
           <bottom>Licenced under the Apache License, Version 2.0</bottom>
+          <additionalOptions>-html5</additionalOptions>
```

Comment:
Is this in some way needed for Java 10 builds?

Reply:
If it is not included there is a ```WARNING``` message as follows 
```
javadoc: warning - You have not specified the version of HTML to use. The default is currently HTML 4.01, but this will change to HTML5 in a future release. To suppress this warning, please specify the version of HTML used in your documentation comments and to be generated by this doclet, using the -html4 or -html5 options.
```
The -html5 additional option cleans up presence of ```WARNING```'s during Maven build.

Reply:
This setting in the parent POM. It is commented out because the build fails with Java8.

1. Is setting in the parent enough? If it does, it is better to have in one place because ...
2. Has this PR been run with Java8? The build exits with an error in jena-iri:javadoc for me.